crates/mysten-network/src/server.rs

```diff
@@ -34,12 +38,23 @@ pub struct ServerBuilder {
 
 type WrapperService = Stack<
     Stack<
-        Either<LoadShedLayer, Identity>,
-        Stack<Either<GlobalConcurrencyLimitLayer, Identity>, Identity>,
+        TraceLayer<SharedClassifier<GrpcErrorsAsFailures>>,
+        Stack<
+            Either<LoadShedLayer, Identity>,
+            Stack<Either<GlobalConcurrencyLimitLayer, Identity>, Identity>,
+        >,
     >,
     Identity,
 >;
 
+// type WrapperService =         S: Service<Request<Body>, Response = Response<BoxBody>, Error = Infallible>
+//             + NamedService
+//             + Clone
+//             + Send
+//             + 'static,
+//         S::Future: Send + 'static,
+//         L: Clone,
+
```

Comment:
Do we want to live this here w/o a comment on what we're trying to achieve?

Reply:
Oops that shouldn't be in here. I'll fix before merging